src/actions/historyActions.js

```diff
@@ -122,6 +123,8 @@ export const fetchSmartWalletTransactionsAction = () => {
       }
     }
 
+    if (!isOnline) return;
```

Comment:
Are those checks necessary? I've counted 4 of them in this PR :p Should we remember about offline check every time we make an api call?

Reply:
Per out last discussion this is needed, however, we can think how we can improve the check throughout the app, however, it's not matching this ticket scope.